packages/airnode-deployer/src/infrastructure/index.ts

```diff
@@ -186,14 +187,14 @@ async function terraformAirnodeManage(
   }
 
   if (httpGateway?.enabled) {
-    commonArguments.push(['var', 'http_api_key', httpGateway.apiKey!]);
+    commonArguments.push(['var', 'http_path_key', randomUUID()]);
```

Comment:
Hmm. The problem with this approach is that the path (the UUID) will change every time the user will redeploy which is quite annoying. I think it would be better to use [`random_uuid`](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/uuid) Terraform resource. That will allow us to create a random path but it will be stored in the Terraform state and therefore won't change between redeployments. Also, we won't need the path variables to pass to the Terraform comamnds.

Reply:
Updated to use this method and tested again that it works.